[2025-09-16T13:50:50.627+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-16T13:50:50.674+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: vars_conns_demo.show_connection manual__2025-09-16T13:50:49.998366+00:00 [queued]>
[2025-09-16T13:50:50.682+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: vars_conns_demo.show_connection manual__2025-09-16T13:50:49.998366+00:00 [queued]>
[2025-09-16T13:50:50.683+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-16T13:50:50.701+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): show_connection> on 2025-09-16 13:50:49.998366+00:00
[2025-09-16T13:50:50.709+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'vars_conns_demo', 'show_connection', 'manual__2025-09-16T13:50:49.998366+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/vars_conn_demo.py', '--cfg-path', '/tmp/tmpgdn2qd38']
[2025-09-16T13:50:50.713+0000] {standard_task_runner.py:91} INFO - Job 14: Subtask show_connection
[2025-09-16T13:50:50.713+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=1415) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-16T13:50:50.714+0000] {standard_task_runner.py:64} INFO - Started process 1416 to run task
[2025-09-16T13:50:50.791+0000] {task_command.py:426} INFO - Running <TaskInstance: vars_conns_demo.show_connection manual__2025-09-16T13:50:49.998366+00:00 [running]> on host 0d34ce0f8ee7
[2025-09-16T13:50:50.910+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='vars_conns_demo' AIRFLOW_CTX_TASK_ID='show_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-09-16T13:50:49.998366+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-16T13:50:49.998366+00:00'
[2025-09-16T13:50:50.911+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-16T13:50:50.924+0000] {base.py:84} INFO - Using connection ID 'demo_conn' for task execution.
[2025-09-16T13:50:50.925+0000] {logging_mixin.py:188} INFO - [CONN] id=demo_conn host=example.com schema= login=kam
[2025-09-16T13:50:50.926+0000] {logging_mixin.py:188} INFO - [CONN] (password is NOT logged)
[2025-09-16T13:50:50.926+0000] {python.py:237} INFO - Done. Returned value was: {'host': 'example.com', 'schema': ''}
[2025-09-16T13:50:50.927+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-16T13:50:50.958+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=vars_conns_demo, task_id=show_connection, run_id=manual__2025-09-16T13:50:49.998366+00:00, execution_date=20250916T135049, start_date=20250916T135050, end_date=20250916T135050
[2025-09-16T13:50:51.009+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-16T13:50:51.059+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-16T13:50:51.066+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
